pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -149,66 +193,167 @@ func createNamespace(deployContext *deploy.DeployContext) (bool, error) {
+			Kind:       "Namespace",
+			APIVersion: corev1.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: DevWorkspaceCheNamespace,
+		},
+		Spec: corev1.NamespaceSpec{},
+	}
+
+	return deploy.CreateIfNotExists(deployContext, namespace)
+}
+
+func syncDwCheServiceAccount(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheServiceAccountFile, &corev1.ServiceAccount{})
+}
+
+func syncDwCheClusterRole(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheClusterRoleFile, &rbacv1.ClusterRole{})
+}
+
+func syncDwCheProxyClusterRole(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheProxyClusterRoleFile, &rbacv1.ClusterRole{})
+}
+
+func syncDwCheMetricsClusterRole(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheMetricsReaderClusterRoleFile, &rbacv1.ClusterRole{})
+}
+
+func syncDwCheLeaderRole(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheRoleFile, &rbacv1.Role{})
+}
+
+func syncDwCheLeaderRoleBinding(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheRoleBindingFile, &rbacv1.RoleBinding{})
+}
+
+func syncDwCheProxyRoleBinding(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheProxyClusterRoleBindingFile, &rbacv1.ClusterRoleBinding{})
+}
+
+func syncDwCheRoleBinding(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheClusterRoleBindingFile, &rbacv1.ClusterRoleBinding{})
+}
+
+func syncDwCheCRD(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheManagersCRDFile, &apiextensionsv1.CustomResourceDefinition{})
+}
+
+func syncDwCheConfigMap(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheConfigMapFile, &corev1.ConfigMap{})
+}
+
+func syncDwCheMetricsService(deployContext *deploy.DeployContext) (bool, error) {
+	return syncObject(deployContext, DevWorkspaceCheMetricsServiceFile, &corev1.Service{})
+}
+
+func synDwCheCR(deployContext *deploy.DeployContext) (bool, error) {
+	// We want to create a default CheManager instance to be able to configure the che-specific
+	// parts of the installation, but at the same time we don't want to add a dependency on
+	// devworkspace-che-operator. Note that this way of initializing will probably see changes
+	// once we figure out https://github.com/eclipse/che/issues/19220
+	obj := &unstructured.Unstructured{}
+	obj.SetGroupVersionKind(schema.GroupVersionKind{Group: "che.eclipse.org", Version: "v1alpha1", Kind: "CheManager"})
+	err := deployContext.ClusterAPI.Client.Get(context.TODO(), client.ObjectKey{Name: "devworkspace-che", Namespace: DevWorkspaceNamespace}, obj)
```

Comment:
we can use IsExists( ... ) from sync.go

Reply:
We can't because `Unstructured` is not `metav1.Object`.